display.go

```diff
@@ -91,6 +205,22 @@ func init() {
 		updateStaticContents()
 		displayInited = true
 		fmt.Println("display inited")
+		wakeDisplay()
 		requestDisplayUpdate()
 	}()
+
+	go func() {
+		// Start display auto-sleeping ticker
+		ticker := time.NewTicker(1 * time.Second)
```

Comment:
Since you are already using a ticker, you might want to consider using https://pkg.go.dev/time#Ticker.Reset to just Reset it back to the DisplayDimAfterMs time whenever there is activity. Would be a bit more efficient and would mean you wouldn't need to have this no-op loop running once every second.

Reply:
Neat, I'll do that.

Reply:
@tutman96 Okay, pushed this - please take a look when you get chance, I'm not super happy with having two methods that are called when the tickers fire (tick_displayDim and tick_displayOff) as they practically do the same job. I wonder if it'd be possible to keep the tickers in an array and figure out which one of them fired, to have one handler for them?